### PR TITLE
Add PHP and Composer into containers

### PR DESCRIPTION
**Pull request by beattie282**

### CODE REVIEW

File: docker/php/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM php:7.3-fpm
```
Comment (richardhughes) [inline docker/php/Dockerfile:1]: Can we use the cli image please?

File: docker/php/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+RUN echo 'Acquire::http::Pipeline-Depth "0";' > /etc/apt/apt.conf.d/00no-pipeline
+ENV TZ=Europe/London
```
Comment (richardhughes) [inline docker/php/Dockerfile]: Don't think we need this lines

File: docker/php/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update && \
```
Comment (richardhughes) [inline docker/php/Dockerfile:3]: Can we install the minimum required to this image please? I think it only needs a few PHP extensions

File: docker/php/config/php.ini [hunk 4]
```diff
@@ -0,0 +1,9 @@
+memory_limit = 256M
```
Comment (richardhughes) [inline docker/php/config/php.ini]: Don't think we need this file, we can just get away with the default PHP ini

File: docker/php/Dockerfile (hunk 1 above)
Comment (beattie282) [reply docker/php/Dockerfile:1]: doing in second pr

File: docker/php/Dockerfile (hunk 2 above)
Comment (beattie282) [reply docker/php/Dockerfile]: removed

File: docker/php/Dockerfile (hunk 3 above)
Comment (beattie282) [reply docker/php/Dockerfile:3]: reduced the list by alot

File: docker/php/config/php.ini (hunk 4 above)
Comment (beattie282) [reply docker/php/config/php.ini]: removed

**Review by richardhughes — APPROVED** [review]

**Pull request merged**